### ISSUE TITLE
allowPageScroll Broken on Iphone

### ISSUE THREAD

**Issue by pvijeh** [mattbryson/TouchSwipe-Jquery-Plugin#267]
this function does not allow native page scrolling functionality on the iphone.  it is broken.

**Comment by arkreddy16** [comment]
Have same issue, any clue?
following is my code snippet:-
 $('div[id="Item"]').each(function () {
                $(this).swipe({
                    swipe: function (event, direction, distance, duration, fingerCount, fingerData) {
                               //Do something
                               }
                    allowPageScroll:"vertical"
                });
            });

**Comment by arkreddy16** [comment]
Got a work arround! used "preventDefaultEvents: false" option.

**Comment by pvijeh** [comment]
I ended up using hammer.js

**Comment by dustin-page** [comment]
allowPageScroll:'auto' or allowPageScroll:'vertical' doesn't allow vertical scrolling on Safari or Chrome browsers on an iPad.

$(window).swipe({
                        swipeRight: function(event, direction, distance, duration, fingerCount, fingerData) {
                            if (fingerData[0].start.x < 50) { //Only "swipeRight" from left edge of screen
                                dnlShow();
                                $('.navbar-collapse').removeClass('in');
                            }

                            $rootScope.$broadcast('p3-global-menu-state', {
                                showMenu: true
                            });
                            scope.$apply();
                        },
                        swipeLeft: function() {
                            $rootScope.$broadcast('p3-global-menu-state', {
                                showMenu: false
                            });
                            scope.$apply();
                        },
                        threshold: 75,
                        allowPageScroll:'vertical'
                    });

**Comment by anupsalpe** [comment]
"preventDefaultEvents: false" is worked for me for vertical pagescroll in iPhone. Thanks @arkreddy16 :)

**Comment by ocala** [comment]
preventDefaultEvents worked also for me. Thanks.

**Comment by mattbryson** [comment]
I'll have a look and see what is causing this....

**Comment by mattbryson** [comment]
Hopefully this is now sorted with c381fc4 .

**Comment by code-smith** [comment]
The new version isnt working but preventDefaultEvents:false is working..!! 
thanks.
@mattbryson  can you please take a look here? is this how its supposed to work? is preventDefaultEvents options a must?

**Comment by mattbryson** [comment]
I'll take a look, it was working fine in the tests on ios. What OS, version
and version of touch swipe are you using?

**Comment by code-smith** [comment]
IOS : both 8 and 9. 
touchswipe: The latest version available in the Github!

I removed allowPageScroll  and set preventDefaultEvents:false and scroll works like a charm.
Btw, i also  raised an issue for latest versions on npm ( last version available on npm is 1.6.12)

**Comment by Verten** [comment]
"preventDefaultEvents: false" is worked for me for vertical pagescroll in iPhone. Thanks @arkreddy16 :)

**Comment by justinputney** [comment]
This problem is still occurring with the vertical pagescroll if you don't add preventDefaultEvents:false.
I've pinpointed the problem to changes that occurred on Oct. 1, 2015 between the following commits:
a29d074 and 90141e8

Tested on iOS 8.3 iPad Simulator.

It looks like it's related to the fingerData variable, since that's where most of the changes occurred.

For now, I can get by with preventDefaultEvents:false, but hopefully this info will help to resolve the problem definitively.
